tasks/main.yml

```diff
@@ -13,6 +13,14 @@
     jenkins_repo_key_url: "{{ __jenkins_repo_key_url }}"
   when: jenkins_repo_key_url is not defined
 
+- name: Create Jenkins user with ssh keys
```

Comment:
This would probably create jenkins user with home directory set to `/home/jenkins`. The apt package would also create this user and set its home directory to `/var/lib/jenkins`. Should we move this task after setup is done? Would that be required? I am not sure. But having home directory set to `/var/lib/jenkins` generally helps when you switch user for debugging.